asf_heat_pump_suitability/analysis/evaluate_hp_suitability_outputs/pandera_eval_hps_outputs.py

```diff
@@ -126,49 +160,77 @@ def add(col: str, dtype, *checks, **kw):
 logging.info("=== Outlier Detection (Z-Score) ===")
 
 Z = cfg.OUTLIER_ZSCORE_THRESHOLD
-num_cols = cfg.NUMERIC_COLUMNS
+logging.info("No outliers detected in any columns (z > %.1f).", Z)
```

Comment:
```suggestion
logging.info("Outlier Z-score threshold: (z > %.1f).", Z)
```
I think this is a mistake?

Reply:
Not sure what happened here in my brain hahaha